### PR TITLE
Updating the spreadsheet control to version 1.1.0 and removed the old files

### PR DESCRIPTION
**Pull request by deaspo**
Updating the spreadsheet control to version 1.1.0 and removed the old files

### CODE REVIEW

**Comment by deaspo** [comment]
Also Fixed the renaming of ucrSelectorForUnstack for the Unstack dialog.

**Comment by dannyparsons** [comment]
Shouldn't there be a change to .vbproj as well here?

File: instat/dlgUnstack.vb [hunk 1]
```diff
@@ -70,9 +70,9 @@ Public Class dlgUnstack
             ucrColumnToUnstackReceiver.RemoveSelected()
         End If
     End Sub
-    Private Sub ucrSelectorByDataFrameAddRemove_KeyPress(sender As Object, e As KeyPressEventArgs) Handles ucrSelectorByDataFrameAddRemove.KeyPress
+    Private Sub ucrSelectorForUnstack_KeyPress(sender As Object, e As KeyPressEventArgs) Handles ucrSelectorForUnstack.KeyPress
         If e.KeyChar = vbCr Then
-            ucrSelectorByDataFrameAddRemove.Add()
+            ucrSelectorForUnstack.Add()
```
Comment (dannyparsons) [inline instat/dlgUnstack.vb]: Thanks for fixing these.But sorry, I also fixed it in my branch to do something. Could you remove the changes here in this pull request so it doesn't conflict with what I've done. Thanks

Comment (deaspo) [inline instat/dlgUnstack.vb]: Sure

**Pull request closed without merging**